components/server/src/user/user-service.ts

```diff
@@ -147,6 +147,8 @@ export class UserService {
         if (!newUser.blocked && (isFirstUser || this.env.makeNewUsersAdmin)) {
             newUser.rolesOrPermissions = ['admin'];
         }
+        // FIXME(janx): Remove this before deploying to production(!)
+        newUser.rolesOrPermissions = (newUser.rolesOrPermissions || []).concat(['teams-and-projects']);
```

Comment:
instead of this, we could whitelist the preview env URLs in the menu.ts for now. I.e. `'teams-and-projects' or preview-env-url`

Reply:
I mean here https://github.com/gitpod-io/gitpod/pull/4401/files#diff-64248da95f399a1cfe99b03aa662caf313faf9b7016999111605da5d61f1971dR43

Reply:
Good idea, done. Also removed the WIP commit.